index.html

```diff
@@ -14,26 +14,164 @@
          
     </nav>
 
+    <section class="landing-page">
+    <div class="all-body" id="body-cover">
+        
+            <h2 class="title">I'm Mariane Glad to see you!</h2>
+            <p class="intro">
+                I'm a software developer! I can help you build a product , 
+                feature or website Look through some of my work and experience! 
+                If you like what you see and have a project you need coded, don't 
+                hestiate to contact me.
+            </p>
+        
+            <p class="lets-connect">LET'S CONNECT</p>
+                
+            <ul class="social-buttons">
+                <li><a href="#"><img src="images/Vector.svg" class="icons"></a></li>
+                <li><a href="#"><img src="images/Group.svg" class="icons"></a></li>
+                <li><a href="#"><img src="images/Vector (3).svg" class="icons"></a></li>
+                <li><a href="#"><img src="images/Vector (4).svg" class="icons"></a></li>
+                <li><a href="#"><img src="images/Vector (5).svg" class="icons"></a></li>
+            </ul>
+        
+    </div>
+    <section>
+        
+    <section class="works-container">
```

Comment:
[OPTIONAL] Good work, implementing the changes suggested by previous reviewer. But I would like to clarify what previous reviewer meant:
- So section is like `landing page` or `my works` or `about me` etc. So all of these would be in their own
```
<section>
   SECTION CODE
</section>
```
Since you have added work and about section in one section tag, kindly create a section for both of them separately.